pysmt/test/test_cnf.py

```diff
@@ -0,0 +1,57 @@
+#
+# This file is part of pySMT.
+#
+#   Copyright 2014 Andrea Micheli and Marco Gario
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+import unittest
+
+import pysmt
+from pysmt.shortcuts import And, Not, Symbol, Bool, Implies, is_sat, is_valid
+from pysmt.cnf import CNFizer
+from pysmt.logics import QF_BOOL, QF_LRA, QF_LIA
+from pysmt.test import TestCase, skipIfNoSolverForLogic
+from pysmt.test.examples import EXAMPLE_FORMULAS
+
+class TestCnf(TestCase):
+
+    def do_examples(self, logic):
+        conv = CNFizer()
+        for example in EXAMPLE_FORMULAS:
```

Comment:
These examples are relatively small. Can we take a (QF) SMT-LIB problem, build the CNF and then check that it yields the same result as the original problem?